src/crawlee/configuration.py

```diff
@@ -233,17 +233,15 @@ class Configuration(BaseSettings):
 
     @classmethod
     def get_global_configuration(cls) -> Self:
-        """Retrieve the global instance of the configuration."""
-        from crawlee import service_container
+        """Retrieve the global instance of the configuration.
 
-        if service_container.get_configuration_if_set() is None:
-            service_container.set_configuration(cls())
+        TODO: Can we remove this?
+        """
+        from crawlee.service_container import get_configuration
 
-        global_instance = service_container.get_configuration()
+        cfg = get_configuration()
```

Comment:
Super tiny detail. In Previous file you called such variable config, but here cfg. So for consistency I would suggest to all it config or cfg everywhere.

Reply:
Makes sense. I will stay with `config` then.